node/src/components/consensus/highway_core/validators.rs

```diff
@@ -76,6 +78,22 @@ impl<VID: Eq + Hash> Validators<VID> {
     pub(crate) fn iter(&self) -> impl Iterator<Item = &Validator<VID>> {
         self.validators.iter()
     }
+
+    /// Marks the validator with that ID as banned, if it exists.
+    pub(crate) fn ban(&mut self, vid: &VID) {
```

Comment:
Wouldn't banning a validator that doesn't exist indicate that it's a programming error? I mean, we should be able to tell whether `vid` exists or not in that era, without "guessing".

Reply:
Currently we just call `ban` for everyone who was accused in any of several recent switch blocks. They might not be validators in this era anymore.

If you prefer, we could filter in the era supervisor, before making this call, of course. I just felt this was simpler, and also kind of correct: If we don't even know that validator, they're effectively banned already anyway.